README.md

```diff
@@ -1,44 +1,71 @@
-- Build and release automation with [ConcourseCI](https://concourse-ci.org/)
+- Build and release automation powered by [ConcourseCI](https://concourse-ci.org/)
 - A private container registry with [Docker Registry](https://docs.docker.com/registry/)
 - Signing of docker image integrity with [Docker Notary](https://docs.docker.com/notary/)
 - Scanning of docker images for security vulnerabilities with [clair](https://github.com/coreos/clair)
 - Monitoring and alerting with [Prometheus](https://prometheus.io/), [Alertmanager](https://prometheus.io/docs/alerting/alertmanager/) and [Grafana](https://grafana.com/)
 - Secure git-based secrets configuration with [sealed-secrets](https://github.com/bitnami-labs/sealed-secrets)
-- Ingress management with [nginx ingress controller](https://kubernetes.github.io/ingress-nginx/)
+- Ingress management and service mesh with [Istio](https://istio.io/)
 - Protective monitoring provided by GDS TechOps CyberSecurity with [Splunk](https://www.splunk.com/)
-- Cloud infrastructure hosted on [AWS](https://aws.amazom.com) in three availability zones in the London region managed with [Terraform](https://www.terraform.io/)
+- Cloud infrastructure hosted on [AWS](https://aws.amazom.com) across multiple availability zones in London
 - Kubernetes control plane with [AWS EKS](https://aws.amazon.com/eks/)
 
+
+## Non goals
+
+GSP is not a managed service.
+
+If you are a team looking for a fully managed platform, we recommend you evaluate [GOV.UK PaaS](https://docs.cloud.service.gov.uk) before attempting to run and manage your own GSP instance. 
+
+The platform has been designed to complement an organisation that practices a Reliability Eningeering model that assumes there exists a small number of infrastructure and reliability focused members capable of supporting a much larger team or programme.
+
+![](./docs/assets/paas-spectrum.png)
+
+The figure above illustrates where we think GSP fits on a "PaaS Spectrum":
+
+* On the right-hand-side we have the situation where service teams all design their deployment architectures in isolation using (hopefully) cloud managed services which gives the ultimate in flexibility at the cost of poor knowledge share accross the organisation and a need for dedicated infrastructure expertise.
+* On the left-hand-side we have the fully managed GOV.UK PaaS platform where a service team may not need any infrastructure expertise but at the cost of flexability.
+* Sitting between the two is GSP which is:
+	* more complex for service teams compared to PaaS, in exchange for greater flexability over the platform itself
+	* more opinionated than a bespoke architecture, in exchange for greater knowledge/code sharing between teams
+	* more isolated than GOV.UK PaaS, but encourages sharing as much as possible 
+
+
 ## Getting started
 
 - [Host a GDS Supported Platform cluster in the AWS cloud](docs/gds-supported-platform/getting-started-gsp-cluster.md)
```

Comment:
Can we change the title of this link so that its consistent with the content it links to

- [Getting started with a GDS Supported Platform cluster](docs/gds-supported-platform/getting-started-gsp-cluster.md)